packages/prisma/zod-utils.ts

```diff
@@ -113,7 +113,9 @@ export type BookingFieldType = typeof BookingFieldType extends z.Values<infer T>
 export const bookingResponses = z
   .object({
     email: z.string(),
-    name: z.string(),
+    name: z.string().refine((value : String) => value.trim().length > 0, {
+      message: "Name is required",
```

Comment:
I don't think we need the message. It's not showing anywhere and 'Invalid Input' is fine as the error message 👍

Reply:
Thank You Carina. I have updated the same.